src/StdinFileInfo.php

```diff
@@ -46,6 +46,8 @@ public function getATime()
 
     public function getBasename($suffix = null)
     {
+        unset($suffix);
```

Comment:
you could put annotation for a code to ignore some rule in some line, but it's putting a tool-related comments into code, which is also not the best ;)